package.json

```diff
@@ -79,7 +79,7 @@
         "style/*.css",
         "style/index.js"
     ],
-    "styleModule": "style/index.js",
+    "style": "style/index.css",
```

Comment:
This is unrelated, but a slight improvement to allow us to delete the `style/index.js` file.